lecture-notes/2-vcs.md

```diff
@@ -1758,22 +1758,114 @@ $
 If you had more remotes, they would also show up both in the configuration file
 and in the output of `git remote -v`.
 
-> As another aside, this convention of the path being `username/projectname` is
-> just that---convention. The server name and path name could be anything that
-> the server chooses. In GitHub's case, though, separating out by username
-> gives nice namespacing (same for the other forges).
+<!-- TODO(max): Figure out x-overflow -->
+<style>
+.sidenote {
+    font-size: 80%;
+    position: relative;
+    float: right;
```

Comment:
Do we want to float this for narrow viewport? Won't that just allow other text to wrap around it even though it should be inline?